src/patternfly/components/CalendarMonth/calendar-month-header-nav.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-calendar-month__header-nav{{#if calendar-month-header-nav--modifier}} {{calendar-month-header-nav--modifier}}{{/if}}"
```

Comment:
could this be `calendar-month-control`

Reply:
I went with nav because it's more specific than a generic control, since we may introduce other controls here. What about `__nav-control`? That would match pagination, and give us room to introduce other types of controls and even a `__nav` in the future if we want a container for the nav.